src/components/RegistrationList.tsx

```diff
@@ -10,17 +10,36 @@ import { displayDate } from '../utils/date-helpers';
 import { TruncatableTypography } from './TruncatableTypography';
 import { ContributorIndicators } from './ContributorIndicators';
 import { SearchListItem } from './styled/Wrappers';
+import EditIcon from '@mui/icons-material/Edit';
+import CloseOutlinedIcon from '@mui/icons-material/CloseOutlined';
+import { ConfirmDialog } from './ConfirmDialog';
+import { useState } from 'react';
+import { deleteRegistration } from '../api/registrationApi';
+import { setNotification } from '../redux/notificationSlice';
+import { isErrorStatus, isSuccessStatus } from '../utils/constants';
+import { getIdentifierFromId } from '../utils/general-helpers';
+import { useDispatch } from 'react-redux';
 
 interface RegistrationListProps {
   registrations: Registration[];
+  canEditRegistration?: boolean;
+  refetchRegistrations?: () => void;
```

Comment:
som diskutert kan vi endre dette til å være `onDeleteDraftRegistration` e.l. og beholde den logikken i `MyRegistrationList` for å unngå å initiere funksjonen en gang for hver gang `RegistrationListItemContent` brukes, da det veldig sjelden faktisk vil være mulig å slette uansett

Reply:
Kan være at ConfirmDialog gjør dette noe mer krøkkete  når jeg tenker meg om... i verste fall kan vi legge inn en sjekk for om brukeren faktisk kan slette før vi lager slett-funksjonen da